tests/unit/amo/pages/TestHome.js

```diff
@@ -199,7 +199,10 @@ describe(__filename, () => {
     expectedThemes.forEach((slug) => {
       expect(
         shelf.find({
-          to: categoryResultsLinkTo({ addonType: ADDON_TYPE_THEME, slug }),
+          to: categoryResultsLinkTo({
+            addonType: ADDON_TYPE_STATIC_THEME,
```

Comment:
This old code also wasn't using `getAddonTypeFilter()`. Is the change needed?

Reply:
Same as above.